db/migrations/0018_add_lti_usage.py

```diff
@@ -0,0 +1,43 @@
+'''
+Migration for MiVideo media started hourly table
+'''
+
+from yoyo import step
+
+steps = [
+    step('''
+        CREATE TABLE IF NOT EXISTS lti_placement_types (
+            id INTEGER NOT NULL AUTO_INCREMENT,
+            description VARCHAR(100) NOT NULL,
+            table_name VARCHAR(64) NOT NULL,
+            PRIMARY KEY (id)
+        ) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4;
+    '''),
+    step('''
+        INSERT IGNORE INTO lti_placement_types
+        (id, description, table_name) VALUES
+        (0, 'Zoom', 'lti_zoom_placement')
+    '''),
+    step('''
+        CREATE TABLE IF NOT EXISTS lti_placement (
+            id BIGINT NOT NULL AUTO_INCREMENT,
+            account_id INTEGER NOT NULL,
+            course_id INTEGER NOT NULL,
+            course_name VARCHAR(200),
+            placement_type INTEGER NOT NULL,
+            PRIMARY KEY (id)
+        ) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4;
+    '''),
+    step('''
+        CREATE TABLE IF NOT EXISTS lti_zoom_placement (
+            id BIGINT NOT NULL AUTO_INCREMENT,
+            placement_id INTEGER NOT NULL,
```

Comment:
placement_id is the FK to lti_placement table, right?

Reply:
Yeah right, I didn't set up FK's